### PR TITLE
[Publisher] Add generic bulk upload templates

### PR DESCRIPTION
**Pull request by mxosman**
## Description of the change

Add generic bulk upload templates.

Now users can toggle between **Custom** or **Generic** templates in the Bulk Upload page. I added a little descriptor underneath the radio buttons to help explain the difference between the two - would love to know your thoughts/any adjustments you'd make.

https://github.com/Recidiviz/justice-counts/assets/59492998/5f40eb4c-d197-4581-8075-0034a86ee332

## Related issues

Closes [#26781](https://github.com/Recidiviz/recidiviz-data/issues/26781)

### Development

**This box MUST be checked by the submitter prior to merging**:
- [x] **Double- and triple-checked that there is no Personally Identifiable Information (PII) being mistakenly added in this pull request**

These boxes should be checked by the submitter prior to merging:
- [ ] Tests have been written to cover the code changed/added as part of this pull request

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: publisher/src/components/DataUpload/InstructionsTemplate.tsx [hunk 1]
```diff
@@ -242,59 +262,77 @@ export const GeneralInstructions: React.FC<
...
-              <DownloadTemplateSystem>
-                {systemName} Single Page
-                <span>Download</span>
-              </DownloadTemplateSystem>
-              {templateKeyToLoadingStatus[`${system}_SINGLE`] === true && (
-                <MiniLoaderWrapper>
-                  <MiniLoader />
-                </MiniLoaderWrapper>
-              )}
-            </DownloadAutogeneratedTemplateBox>
```
Comment (mxosman) [inline publisher/src/components/DataUpload/InstructionsTemplate.tsx:297]: Here, I'm consolidating these two similar `DownloadAutogeneratedTemplateBox` into one

**Comment by mxosman** [comment]
Hi @jwhrec! I was going to deploy this to playtesting, but wanted to give you a chance to play with Brandon's new feature incase you wanted to!

Here's how you would deploy this branch to playtesting:
1. Click on the **Actions** tab in Github 

Screenshot 2024-02-01 at 4 58 06 PM

2. Click on **Deploy to Playtest** on the left side bar
3. Click on the **Run workflow** drop down on the right side
4. Under **User workflow from** type in this branch: `mahmoud/update-bulk-upload-templates`

Screenshot 2024-02-01 at 4 59 13 PM

5. Last but not least, type in `mahmoudtest` for the **Playtest URL tag** and click on **Run workflow**

Screenshot 2024-02-01 at 4 59 25 PM

6. A cool bot will drop the link in the PR, but give it like ~9 min to fully deploy after the link posts.

**Comment by helperbot-recidiviz** [comment]
@jwhrec successfully triggered a playtest deployment. Full deployment usually takes 5 minutes. Your playtest link is https://mahmoudtest---publisher-web-b47yvyxs3q-uc.a.run.app/

**Review by jwhrec — APPROVED** [review]
checked it out in playtesting, and things look great to me!

**Comment by mxosman** [comment]
Thank you, Jessica!

**Pull request merged**